torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -551,6 +551,45 @@ std::vector<at::Tensor> FusionExecutor::runFusion(
   return allocated_outputs;
 }
 
+void FusionExecutor::compileRTC(
```

Comment:
naming: `compileRtc` and `runRtc`

https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Reply:
updated. Thanks!